public/javascripts/items.js

```diff
@@ -56,9 +55,8 @@ const titleEdited = event => {
 const descriptionEdited = event => {
   const elementToHide = $( event.target )
   const id = elementToHide.data( 'id' )
-  const elementToShow = $( selector( 'description', id, 'span' ) )
-
-  if( event.charCode === RETURN_KEY ) {
+  const elementToShow = $( selector( 'item__description', id, 'span' ) )
+  if( event.keyCode === RETURN_KEY ) {
```

Comment:
Please add whitespace back - there *should* be whitespace between the setup of a function, and the logic.